near/contracts/bridge/src/lib.rs

```diff
@@ -375,6 +391,75 @@ impl FastBridge {
         U128::from(0)
     }
 
+    #[pause(except(roles(Role::UnrestrictedUnlock)))]
```

Comment:
Why did we choose `Unrestricted Unlock` not `Unrestricted Withdraw`? Should we create a special role for this function?

Reply:
ec6a4f2ed7b5f4ae6d9f38f0b393202c773f4593